codebuild/release/validate.yml

```diff
@@ -21,7 +21,7 @@ phases:
   pre_build:
     commands:
       - cd examples
-      - sed -i "s/dynamodb-encryption-sdk/dynamodb-encryption-sdk==$VERSION/" ../../dev_requirements/test-requirements.txt
+      - sed -i "s/dynamodb-encryption-sdk/dynamodb-encryption-sdk==$VERSION/" ../dev_requirements/examples-requirements.txt
```

Comment:
This `sed` command seems like it probably breaks with the pinning of dependencies. When I try it locally I get:
```
dynamodb-encryption-sdk====3.1.0
```